acceptance/tests/command.rb

```diff
@@ -0,0 +1,15 @@
+test_name "C100546: bolt command run should execute command on remote hosts\
+           via ssh" do
+  step "execute `bolt command run` via SSH" do
+    ssh_nodes = select_hosts(roles: ['ssh'])
+    nodes_csv = ssh_nodes.map(&:hostname).join(',')
+    command = 'hostname -f'
+    bolt_command = "bolt command run --nodes #{nodes_csv} '#{command}'"
+    case bolt['platform']
+    when /windows/
+      execute_powershell_script_on(bolt, bolt_command)
```

Comment:
I think this starts a new `powershell.exe` interpreter each time, which is pretty slow to run a single command. Can we just do `on(bolt, "cmd /c #{bolt_command}")`? /cc @Iristyle

Reply:
Yes that starts a new `powershell.exe` and yes it's slow...

However, it also simulates closer to what we expect end users to be doing, which may impact argument passing conventions, etc.  I would assume that's desireable here?

Reply:
The desire is to simulate what we have instructed the user to perform. I think this performance hit is reasonable to endure at the moment. If it is determined not to add value later it can be changed.